typescript/sdk/src/ism/types.ts

```diff
@@ -275,6 +296,10 @@ export const PausableIsmConfigSchema = PausableSchema.and(
   }),
 );
 
+export const IcaIsmConfigSchema = z.object({
+  type: z.literal(IsmType.ICA),
+});
```

Comment:
why have config for this at all?

Reply:
https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/6255/files#r2125061738